lib/src/module.dart

```diff
@@ -16,68 +16,4 @@ library w_module.src.module;
-/// - The [components] class exposes react-dart compatible UI components that
-///   can be used to render module data.
-abstract class Module extends LifecycleModule {
-  /// The [api] object should contain all public methods that a consumer can use
-  /// to mutate module state (methods) or query existing module state (getters).
-  ///
-  /// [api] is initially null.  If a module exposes a public [api], this should
-  /// be overridden to provide a class defined specifically for the module.
-  ///
-  /// If using with w_flux internals, module mutation methods should usually
-  /// dispatch existing actions available within the module.  This ensures
-  /// that the internal unidirectional data flow is maintained, regardless of
-  /// the source of the mutation (e.g. external api or internal UI).  Likewise,
-  /// module methods that expose internal state should usually use existing
-  /// getter methods available on stores within the module.
-  Object get api => null;
-
-  /// The [components] object should contain all react-dart compatible UI
-  /// component factory methods that a consumer can use to render module data.
-  ///
-  /// [components] is initially null.  If a module exposes public [components],
-  /// this should be overridden to provide a class defined specifically for the
-  /// module.  By convention, the custom [components] class should extend
-  /// [ModuleComponents] to ensure that the default UI component is available
-  /// via the module.components.content() method.
-  ///
-  /// If using with w_flux internals, [components] methods should usually return
-  /// UI component factories that have been internally initialized with the
-  /// proper actions and stores props.  This ensures full functionality of the
-  /// [components] without any external exposure of the requisite internal
-  /// actions and stores.
-  ModuleComponents get components => null;
-
-  /// The [events] object should contain all public streams that a consumer can
-  /// listen to for notification of internal module state change.
-  ///
-  /// [events] is initially null.  If a module exposes public [events], this
-  /// should be overridden to provide a class defined specifically for the
-  /// module.
-  ///
-  /// If using with w_flux internals, [events] should usually be dispatched by
-  /// internal stores immediately prior to a corresponding trigger dispatch.
-  /// [events] should NOT be dispatched directly by UI components or in
-  /// immediate response to actions.  This ensures that the internal
-  /// unidirectional data flow is maintained and external [events] represent
-  /// confirmed internal state changes.
-  Object get events => null;
-}
-
-/// Standard [Module.components] class. If a module implements a custom class
-/// for its components, it should extend [ModuleComponents].
-abstract class ModuleComponents {
-  /// The default UI component
-  Object content() => null;
-}
+abstract class Module extends LifecycleModule {}
```

Comment:
Hmm. This class no longer has much utility. @maxwellpeterson-wf do you think it might be worth doing some deprecation gymnastics so that we can reclaim the `Module` name and use it in place of `SimpleModule` in the future?